app/logic/events.py

```diff
@@ -214,7 +214,7 @@ def calculateRecurringEventFrequency(event):
         raise Exception("This event is not a recurring event")
 
     return [ {'name': f"{event['name']} Week {counter+1}",
-              'date': event['startDate'] + datetime.timedelta(days=7*counter),
+              'date': (event['startDate'] + datetime.timedelta(days=7*counter)).strftime("%m/%d/%Y"),
```

Comment:
This change is test-breaking. Make sure it doesn't break the existing application.

Reply:
Done